superset-frontend/plugins/plugin-chart-echarts/src/utils/helper.ts

```diff
@@ -0,0 +1,4 @@
+export const convertInteger = (value: string | number) => {
```

Comment:
1. You need to add a license to make this PR pass the CI (just copy paste the comment from top of any js file)
2. Can we we use more descriptive file names? Maybe the same as the function name

Reply:
@prosdev0107 checking in on these last little tweaks... otherwise, I think this one is looking good!